README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# Homework 01
+
+Hello world! My name is Marc-Antoine Manzoni.
+I'm a second (soon to be third year) in *The College*.
+I am majoring in History, and minoring in East Asian Languages and Cultures (EALC). Please enjoy this picture of me and a very nice dog I met recently.
+![this picture](Images/4fc3f091-bd9b-4edb-9f5a-34505d985046.JPG). 
```

Comment:
If you want these to be separate list items, use list syntax. If you want them to actually be separate paragraphs, you need to include a blank line between the text